api/pom.xml

```diff
@@ -147,25 +166,36 @@
                         <artifactId>maven-compiler-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>jdk-9</id>
+                                <id>jdk-11</id>
                                 <goals>
                                     <goal>compile</goal>
                                 </goals>
                                 <configuration>
-                                    <source>9</source>
-                                    <target>9</target>
+                                    <source>11</source>
+                                    <target>11</target>
                                     <includes>
                                         <include>module-info.java</include>
                                     </includes>
                                 </configuration>
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <configuration>
+                            <detectJavaApiLink>false</detectJavaApiLink>
+                            <links>
+                                <link>https://docs.oracle.com/en/java/javase/11/docs/api</link>
```

Comment:
is this needed?

Reply:
Instead of pointing to an oracle website for javadoc, we could instead point to javadoc.io which automatically hosts javadoc produced by github projects, for example:
https://javadoc.io/doc/jakarta.json.bind/jakarta.json.bind-api

Reply:
I'll replace it with ```<detectOfflineLinks>false</detectOfflineLinks>```